release/_hotfix.sh

```diff
@@ -138,7 +138,7 @@ function compare_jars {
 			grep --invert-match "__liferay__/index.js" | \
 			grep --invert-match "_jsp.class" | \
 			grep --invert-match "_jsp.java" | \
-			grep --invert-match "_jsp\$1.class" | \
+			grep --invert-match "_jsp\$[0-9]\+\.class"
```

Comment:
@kiwm, why did you need to replace `.class` by `\.class`?

Was it necessary to fix the bug reported on LPD-70998?

Thanks in advance